src/test/java/pageObject/HomePage.java

```diff
@@ -0,0 +1,65 @@
+package pageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+public class HomePage extends BasePage {
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By titleLocator = By.cssSelector("title");
+    By playButtonLocator = By.cssSelector("[data-testid='play-btn']");
+    By pauseButtonLocator = By.cssSelector("[data-testid='pause-btn']");
+    By allSongsLocator = By.cssSelector(".songs");
+    By userPlaylistElement3 = By.cssSelector(".playlist:nth-child(3)");
+    By userPlaylistInputField = By.cssSelector("input[name='name']");
+    By successBanerLocator = By.cssSelector(".success");
```

Comment:
**I recommend:**  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");